examples/movie_view_ratings_local.py

```diff
@@ -0,0 +1,76 @@
+""" Demo of PipelineDP with Apache Beam.
+"""
+
+from absl import app
+from absl import flags
+import pipeline_dp
+
+from examples.example_utils import parse_file, write_to_file
+
+FLAGS = flags.FLAGS
+flags.DEFINE_string('input_file', None, 'The file with the movie view data')
+flags.DEFINE_string('output_file', None, 'Output file')
+
+
+def main(unused_argv):
+    # Here, we use a local backend for computations. Which does not depend on
+    # any pipeline framework and it is implemented in pure Python in
+    # PipelineDP. It keeps all data in memory and not optimized for large data.
+    # For datasets till tens of megabytes it is faster than running Beam or Spark
```

Comment:
For datasets smaller than ~tens of megabytes, local execution without any framework is faster than local mode with Beam or Spark.